Resources/Prototypes/Loadouts/loadout_groups.yml

```
@@ -966,14 +966,14 @@
   loadouts:
   - HeadofSecurityCloak
   - HeadofSecurityMantle
+  - Bodycam # Goobstation - Body cameras
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)